.github/workflows/release.yml

```diff
@@ -46,32 +46,30 @@ jobs:
           distribution: 'temurin'
           cache: 'maven'
 
+      # Need Maven 3.9.0+ to recognize MAVEN_ARGS.
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.5
+        with:
+          maven-version: 3.9.3
+
       # We skip tests in dry run, to make the process faster.
       # Technically, this goes against the main reason for doing a dry run; to eliminate potential problems.
       # But unless something catastrophic happened, PR checks on source branch already ensured that all tests pass.
       - name: Set release version and build release
         run: |
           mvn -Dfull versions:set -DnewVersion=${{ github.event.inputs.version }}
           mvn -Dfull deploy -DskipTests=${{ github.event.inputs.dryRun }} -DaltDeploymentRepository=local::default::file://`pwd`/target/staging-deploy
-
-      - name: Update release branch and commit the changes
-        run: |
           cp docs/target/antora-template.yml docs/src/antora.yml
           git add docs/src/antora.yml
           find . -name 'pom.xml' | xargs git add
           git commit -m "chore: release version ${{ github.event.inputs.version }}"
-
-      # JReleaser requires the release branch to exist, so we need to push it before releasing.
-      # Once this is pushed, branch protection rules apply.
-      # So if any of the subsequent steps should fail, the release branch is there to stay; cannot be deleted.
-      # To minimize that chance, do a dry run first, with a branch named in a way that the protection rules don't apply.
-      - name: Push release branch to Git
-        run: |
           git push origin ${{ github.event.inputs.releaseBranch }}
 
       - name: Run JReleaser
-        run: mvn -Dfull -Djreleaser.dry.run=${{ github.event.inputs.dryRun }} -pl . jreleaser:full-release
+        uses: jreleaser/release-action@v2
```

Comment:
There is a dedicated GHA? Interesting.

Reply:
Yes. I'm already using it in `enterprise`, so I'm just making this consistent with that.